src/dotnet/commands/dotnet-install/dotnet-install-tool/InstallToolCommandParser.cs

```diff
@@ -19,6 +19,10 @@ public static Command InstallTool()
                     "-g|--global",
                     LocalizableStrings.GlobalOptionDescription,
                     Accept.NoArguments()),
+                Create.Option(
+                    "--tool-path",
+                    "Location of shim to access tool",
```

Comment:
Perhaps `"Location to install the tool to."` (loc'd of course).

Reply:
Yours are better. But we haven't found a good message yet. Need to continue 

KathleenDollard 3 days ago  Member
More like:

**"Location of shim to access tool, allowing unique path for each shell session."**

I don't love this, I'm still working on it because I hate that we are not explaining that it is a temporary as in vulnerable installation.

  @nguerrera
nguerrera 3 days ago  •  Member
But why is it temporary and vulnerable? I think we need to stop thinking in time dimension ("sessions") and just worry about space (global location vs. explicit location vs. default local location). If I can control where, I can control when to cleanup separately. "sessions" becomes a pattern (install to local location, use, uninstall from local location), but not a primitive concept to the CLI.

   @wli3
wli3 23 hours ago  Member
@KathleenDollard @nguerrera In implementation it does have nothing to do with sessions. But the goal of CI author is to achieve isolation from different run. Maybe Location of shim to access tool. Different path can isolate different tool sets.

   @wli3
wli3 23 hours ago  Member
Do you think it is ok to give a paragraph here? This is suitable for automation scenario, recommend each automation run to have a unique PATH so there is no interact with different CI run.... The good part is people don't need to look up for document. And bad part is if we are wrong, we stuck with it

  @nguerrera
nguerrera 23 hours ago  Member
I'd rather we just say what it does, which is much more general than these scenarios. It seems obvious to me why one might pick a unique location. To me, this is like saying man cp should say **"pick a unique destination if you don't want to share the copied file with other things."**

Reply:
"Override default location"? 

Override??

Also we need to find a way to tell user can call your tools with `%toolpath%mytool`